README.md

```diff
@@ -738,22 +738,101 @@ module('Acceptance | super secret url', function(hooks) {
 
 ### ember-cli-qunit 4.1.0 and earlier
 
-For existing apps, the test helpers are merged into your applications namespace,
-and can be imported from the `helpers/ember-simple-auth` module:
+
+For apps using earlier versions of ember-cli-qunit, you can use the
+test helpers with the following signature:
+
+* `currentSession(app)`: returns the current session of your test application.
+* `authenticateSession(app, sessionData)`: authenticates the session; the
+  optional `sessionData` argument can be used to mock an authenticator
+  response - e.g. a token.
+* `invalidateSession(app)`: invalidates the current session in your test application.
+
+For existing apps, the test helpers are merged into your application's namespace,
+and can be imported from the `helpers/ember-simple-auth` module like this:
 
 ```js
 // tests/acceptance/…
 import { currentSession, authenticateSession, invalidateSession } from '<app-name>/tests/helpers/ember-simple-auth';
 ```
 
-**Note** These helpers require the `app` argument.
+The test helpers used in apps using ember-cli-qunit 4.1.0 and earlier all require access to the test application instance.
 
-These helpers have the following function signatures:
-* `currentSession(app)`: returns the current session.
-* `authenticateSession(app, sessionData)`: authenticates the session; the
-  optional `sessionData` argument can be used to mock an authenticator
-  response - e.g. a token.
-* `invalidateSession(app)`: invalidates the session.
+Pass in your application instance as a first parameter to the test helper functions to
+get a handle on your application's session store. 
+You can create an application instance using Ember's `startApp` test helper:
+
+```js
+import startApp from '../helpers/start-app';
+
+let app = startApp();
+```
+
+Make sure to destroy your app instance after each test case. 
+In an `ember-qunit` test suite this will look as follows:
+
+```js
+import startApp from '../helpers/start-app';
+
+let app;
+
+moduleForAcceptance('Acceptance | authentication', {
+  beforeEach: function () {
+    app = startApp();
```

Comment:
I just realized it might be easier if we just use `this.application` which is automatically generated 
 in`moduleForAcceptance` by default. Then we can omit the full explanation of all of the application setup in this section then